example/src/components/Reader.tsx

```diff
@@ -22,7 +22,7 @@ export const Reader: React.FC = () => {
   const [file, setFile] = useState<File>();
   const [location, setLocation] = useState<Locator | Link>();
   const [settings, setSettings] = useState<Partial<Settings>>(DEFAULT_SETTINGS);
-  const ref = useRef<any>();
+  const ref = createRef<BaseReadiumViewRef>();
```

Comment:
[`createRef` should not be used with functional components](https://react.dev/reference/react/createRef).

Reply:
I  used createRef to fix a typescript linting error. I'll look into it again

Reply:
Fixed